.gitignore

```diff
@@ -6,4 +6,5 @@ test/output/
 examples/example-options.json
 .idea
 /composer.phar
-test.php
\ No newline at end of file
+test.php
+composer.lock
```

Comment:
gotcha... It looks like it's just failing for 5.5. We follow the PHP supported versions (http://php.net/supported-versions.php) and 5.5 is EOL. Let's just drop the 5.5 req from travis.yml, that should mean we can keep the lock file.